### DISCUSSION TITLE
Enrollment Issue with Anadolu University Distance Education Program

### DISCUSSION THREAD

**Discussion by atomdeniz** [community/community#146494] [GitHub Education]
### Hi everyone,

I’m currently trying to enroll in a program, but I’ve run into an issue. I’m a student at Anadolu University, pursuing distance education. However, there’s no official document or formal registration certificate available for this program. The only proof I have is the information listed on the university’s official website under the “About” section. Unfortunately, this isn’t being accepted as sufficient documentation by the institution I’m applying to. Also I don't forget to add my student certificate.

Has anyone faced a similar problem? If so, how did you resolve it? Any advice or suggestions on how to handle this situation would be greatly appreciated.

Thank you in advance for your help!

**Comment by amertgokce** [comment]
I'm also having the same problem, it instantly declines my application

**Comment by TomoNx** [comment]
me to, i have same problem
